airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/schemas/calls.json

```diff
@@ -21,7 +21,7 @@
       "type": ["null", "number"]
     },
     "phone_number": {
-      "type": ["null", "number"]
+      "type": ["null", "string"]
```

Comment:
The API says `phone_number` is string https://developer.zendesk.com/api-reference/sales-crm/resources/calls/ Do you have another info @speckm ?

Reply:
The API says `number` not `string`.  This seems to be a documentation error. The data which you get from a request includes strings. Here is a excerpt from a call to the api (made 2023-08-27):

```
{
    id: 134328917,
    user_id: 6907886,
    summary: 'Energieausweis auf H, 3 WE, Haus frisch gekauft',
    recording_url: null,
    outcome_id: null,
    duration: null,
    phone_number: '+491762203????',
    incoming: false,
    missed: false,
    updated_at: '2023-08-18T08:46:33.000Z',
    external_id: null,
    associated_deal_ids: [ 159638??? ],
    made_at: '2023-08-18T08:46:33Z',
    resource_id: 473697140,
    resource_type: 'contact'
  },
  {
    id: 134328735,
    user_id: 6907886,
    summary: null,
    recording_url: null,
    outcome_id: 4974167,
    duration: null,
    phone_number: '+49171321????',
    incoming: false,
    missed: false,
    updated_at: '2023-08-18T08:40:49.000Z',
    external_id: null,
    associated_deal_ids: [ 159420??? ],
    made_at: '2023-08-18T08:40:49Z',
    resource_id: 473105065,
    resource_type: 'contact'
  },
```

I switched some number to ? to keep data save.